test/package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@hugwuoke Somehow this snuck in here. Just a short 6 line file. But it's no problem. I will delete!